src/components/MiniCalendar.vue

```
@@ -0,0 +1,273 @@
+  <div class="mini-calendar">
+    <div class="mini-calendar-header">
+      <button @click="prevMonth" class="nav-button prev-month" aria-label="Previous month">&lt;</button>
+      <span class="month-year">{{ formattedMonthYear }}</span>
+      <button @click="nextMonth" class="nav-button next-month" aria-label="Next month">&gt;</button>
+    </div>
+    <div class="mini-calendar-weekdays">
+      <span v-for="weekday in weekdays" :key="weekday">{{ weekday }}</span>
+    </div>
+    <div class="mini-calendar-grid">
+      <div
+        v-for="(day, index) in daysGrid"
```

ESLint

[error] 'index' is defined but never used.

(vue/no-unused-vars)